src/nft/components/explore/Banner.tsx

```diff
@@ -0,0 +1,175 @@
+          <Box style={{ height: '386px' }}>
+            <div
+              className={styles.bannerWrap}
+              style={{ backgroundImage: `url(${collections[current].bannerImageUrl})` }}
+            >
+              <Box className={styles.bannerOverlay} width="full" />
+              <CollectionDetails collection={collections[current]} hovered={hovered} rank={current + 1} />
+              <CarouselProgress length={collections.length} currentIndex={current} setCurrent={setCurrent} />
+            </div>
+          </Box>
+        </Link>
+      ) : (
+        <>
+          {/* TODO: Improve Loading State */}
+          <p>Loading</p>
+        </>
+      )}
+    </Box>
+  )
+}
+
+export default Banner
+
+/* Collection Details: displays collection stats within Banner  */
+const CollectionDetails = ({
+  collection,
+  rank,
+  hovered,
+}: {
+  collection: TrendingCollection
+  rank: number
+  hovered: boolean
+}) => (
+  <Box as="section" className={styles.section} paddingTop="40">
+    <Column className={styles.collectionDetails} paddingTop="24">
+      <div className={styles.volumeRank}>#{rank} volume in 24hr</div>
+      <Row>
+        <Box as="span" marginTop="16" className={clsx(header1, styles.collectionName)}>
+          {collection.name}
+        </Box>
+        {collection.isVerified && (
+          <Box as="span" marginTop="24">
+            <VerifiedIcon height="32" width="32" />
+          </Box>
+        )}
+      </Row>
+      <Row className={bodySmall} marginTop="12" color="explicitWhite">
+        <Box>
+          <Box as="span" color="darkGray" marginRight="4">
+            Floor:
+          </Box>
+          {collection.floor ? formatEthPrice(collection.floor.toString()) : '--'} ETH
+        </Box>
+        <Box>
+          {collection.floorChange ? (
+            <Box as="span" color={collection.floorChange > 0 ? 'green200' : 'error'} marginLeft="4">
+              {collection.floorChange > 0 && '+'}
+              {formatChange(collection.floorChange)}%
+            </Box>
+          ) : null}
+        </Box>
+        <Box marginLeft="24" color="explicitWhite">
+          <Box as="span" color="darkGray" marginRight="4">
+            Volume:
+          </Box>
+          {collection.volume ? putCommas(+toSignificant(collection.volume.toString())) : '--'} ETH
+        </Box>
+        <Box>
+          {collection.volumeChange ? (
+            <Box as="span" color={collection.volumeChange > 0 ? 'green200' : 'error'} marginLeft="4">
+              {collection.volumeChange > 0 && '+'}
+              {formatChange(collection.volumeChange)}%
+            </Box>
+          ) : null}
+        </Box>
+      </Row>
+      <Link
+        className={clsx(buttonMedium, styles.exploreCollection)}
+        to={`/nfts/collection/${collection.address}`}
+        style={{ textDecoration: 'none', backgroundColor: `${hovered ? vars.color.blue400 : vars.color.grey700}` }}
```

Comment:
is this link hover behavior unique to this one usage?

Reply:
same as above in this case - From a glance at Figma this button will no longer exist in the next iteration.